reinforces reinforced walls attempt 2 [MDB IGNORE]

BOBAMAx opened the pull request:
# About the pull request

redo of #9312

now also ensures all mapper icons signify which walls are hull walls and reinforced walls at a glance.
also makes some of the walls in solaris garage not reinforced so that one guy stops complaining about making reinforced walls actually work as intended

# Explain why it's good for the game
# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>


# Changelog
:cl:
fix: Mapped in reinforced walls are now actually reinforced, and have the same properties as reinforced walls made by hand with plasteel.
/:cl:

BOBAMAx committed:
test

BOBAMAx committed:
whoops

BOBAMAx committed:
Update wall_types.dm

BOBAMAx committed:
icons

BOBAMAx committed:
Update BigRed.dmm

BOBAMAx committed:
Merge remote-tracking branch 'upstream/master' into r_walls2

BOBAMAx committed:
Update USS_Almayer.dmm

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

BOBAMAx committed:
Merge remote-tracking branch 'upstream/master' into r_walls2

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

BOBAMAx committed:
Update LV759_Hybrisa_Prospera.dmm

BOBAMAx committed:
Fixup maps in TGM format

438612fc0358: maps/map_files/LV759_Hybrisa_Prospera/LV759_Hybrisa_Prospera.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Nanu308 committed:
Merge branch 'master' into r_walls2

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

BOBAMAx committed:
Merge remote-tracking branch 'upstream/master' into r_walls2

BOBAMAx committed:
updatepaths

BOBAMAx committed:
Fixup maps in TGM format

f40777f8aecb: maps/map_files/Kutjevo/Kutjevo.dmm
f40777f8aecb: maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm
f40777f8aecb: maps/map_files/Sorokyne_Strata/standalone/clfcamp.dmm

Automatically commited by: tools\mapmerge2\fixup.py

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

BOBAMAx committed:
Merge remote-tracking branch 'upstream/master' into r_walls2

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

BOBAMAx committed:
Merge remote-tracking branch 'upstream/master' into r_walls2

BOBAMAx committed:
Update clfsmugglers.dmm

Nanu308 reviewed (approved):
tldr. fine, but there might been spots that are troublesome that have been missed

Detective-Google commented:
I still agree this is a good idea, but this PR has been conflicted for 7 months now. Feel free to request reopen/make a new PR if you decide to pick it back up.
